path_config.go

```diff
@@ -178,9 +217,29 @@ func (b *azureAuthBackend) pathConfigDelete(ctx context.Context, req *logical.Re
 	return nil, err
 }
 
+func (b *azureAuthBackend) saveConfig(ctx context.Context, config *azureConfig, s logical.Storage) error {
+	entry, err := logical.StorageEntryJSON(configStoragePath, config)
+	if err != nil {
+		return err
+	}
+
+	err = s.Put(ctx, entry)
+	if err != nil {
+		return err
+	}
+
+	// reset the backend since the client and provider will have been
+	// built using old versions of this data
+	b.reset()
+
+	return nil
+}
+
 const (
-	confHelpSyn  = `Configures the Azure authentication backend.`
-	confHelpDesc = `
+	defaultRootPasswordTTL = 4380 * time.Hour
```

Comment:
I think it'd be good to preserve [this comment](https://github.com/hashicorp/vault-plugin-secrets-azure/blob/main/path_config.go#L19-L21) from Azure secrets.

Reply:
Added in latest commit